rest/config.go

```diff
@@ -719,8 +719,16 @@ func (config *ServerConfig) deprecatedConfigLoggingFallback() (warnings []base.D
 			warnings = append(warnings, func() {
 				base.Warnf(warningMsgFmt, `logging.["default"].LogFilePath`, "logging.log_file_path")
 			})
+
 			// Set the new LogFilePath to be the directory containing the old logfile, instead of the full path.
-			config.Logging.LogFilePath = filepath.Dir(*config.Logging.DeprecatedDefaultLog.LogFilePath)
+			// SGCollect relies on this path to pick up the standard and rotated log files.
+			info, err := os.Stat(*config.Logging.DeprecatedDefaultLog.LogFilePath)
+			if err == nil && info.IsDir() {
+				config.Logging.LogFilePath = *config.Logging.DeprecatedDefaultLog.LogFilePath
+			} else {
+				base.Infof(base.KeyAll, "Writing logs into the parent directory of the path specified")
```

Comment:
This log line has a few problems:
- it doesn't tell you what directory is being used
- 'the path specified' doesn't give any hint that it's talking about the deprecated default logging logFilePath
Including both of those pieces of information makes this logging much more useful as a diagnostic aid

I'd suggest moving the logging after we set config.Logging.LogFilePath and change to something like:
```suggestion
				base.Infof(base.KeyAll, "Using %v as log file path (parent directory of deprecated logging.["default"].LogFilePath)", config.Logging.LogFilePath)
```